README.md

```diff
@@ -1,3 +1,24 @@
+# Harvest4Us Version 0.1.0
+**Harvest4Us is an android application that connects local farmers to their community, making it easy for consumers to buy fresh produce.**
+
+## Release Notes
+
+
+### **Version 0.1.0 Features**
+* **Implemented new UI and color scheme**
+* **Added an educational resources page**
+* **transitioned old database from boto3 framework to Flask**
```

Comment:
This should say "backend" rather than "database". You can also add that we started the AWS RDS MySQL database.